pacgraph: add page (#13499)
tldr-pages/tldr: pages/linux/pacgraph.md

# pacgraph

> > Draw a graph of installed packages to PNG/SVG/GUI/console. Good for finding bloat.

Reviewer: Is this part required?

```suggestion
> Draw a graph of installed packages to PNG/SVG/GUI/console.
```

> > Draws a graph of installed packages to PNG/SVG/GUI/console. Good for finding bloat.

Reviewer 2: ```suggestion
> Draw a graph of installed packages to PNG/SVG/GUI/console. Good for finding bloat.
```

> More information: <https://github.com/keenerd/pacgraph>.

> - Produce SVG and PNG graph:

Reviewer: ```suggestion
- Produce a SVG and PNG graph:
```

`pacgraph`

> - Produce SVG graph:

Reviewer: ```suggestion
- Produce a SVG graph:
```

Author: Can you change the `a` articles to `an`?

Reviewer: Done

`pacgraph --svg`

- Print summary to console:

`pacgraph --console`

> - Override default filename/location. Do not specify extension:

Reviewer: ```suggestion
- Override the default filename/location (Note: Do not specify the file extension):
```

> `pacgraph --file=FILENAME`

Reviewer 2: ```suggestion
`pacgraph --file={{path/to/file}}`
```
See style guide and contributing.md

- Change color of packages which are not dependencies:

> `pacgraph --top=COLOR`

Reviewer 2: ```suggestion
`pacgraph --top={{color}}`
```

- Change color of package dependencies:

> `pacgraph --dep=COLOR`

Reviewer 2: ```suggestion
`pacgraph --dep={{color}}`
```

- Change background color of a graph:

> `pacgraph --background=COLOR`

Reviewer 2: **
```suggestion
`pacgraph --background={{color}}`
```
**

- Change color of links between packages:

> `pacgraph --link=COLOR`

Reviewer 2: ```suggestion
`pacgraph --link={{color}}`
```

> > Draw a graph of installed packages to PNG/SVG/GUI/console. Good for finding bloat.
> > More information: <https://github.com/keenerd/pacgraph>.
> 
> - Produce svg and png graph:
> 
> `pacgraph`
> 
> - Produce svg graph:
> 
> `pacgraph --svg`
> 
> - Print summary to console:
> 
> `pacgraph --console`
> 
> - Override default filename/location. Do not specify extension:
> 
> `pacgraph --file={{path/to/file}}`
> 
> - Change color of packages which are not dependencies:
> 
> `pacgraph --top={{color}}`
> 
> - Change color of package dependencies:
> 
> `pacgraph --dep={{color}}`
> 
> - Change background color of a graph:
> 
> `pacgraph --background={{color}}`
> 
> - Change color of links between packages:
> 
> `pacgraph --link={{color}}`

Reviewer 2: ```suggestion
> Draw a graph of installed packages to PNG/SVG/GUI/console. Good for finding bloat.
> More information: <https://github.com/keenerd/pacgraph>.

- Produce SVG and PNG graph:

`pacgraph`

- Produce SVG graph:

`pacgraph --svg`

- Print summary to console:

`pacgraph --console`

- Override default filename/location. Do not specify extension:

`pacgraph --file={{path/to/file}}`

- Change the color of packages that are not dependencies:

`pacgraph --top={{color}}`

- Change the color of package dependencies:

`pacgraph --dep={{color}}`

- Change the background color of a graph:

`pacgraph --background={{color}}`

- Change the color of links between packages:

`pacgraph --link={{color}}`
```
